static/sass/_pattern_cube-table.scss

```diff
@@ -105,9 +105,20 @@
   }
 }
 
-.p-table--cube__badge {
+.p-table--cube__badge-column {
+  @media only screen and (min-width: $breakpoint-medium) {
+    padding-left: 0;
+    padding-right: 0;
+  }
   @media only screen and (max-width: $breakpoint-medium) {
-    margin-top: -0.4rem !important;
+    align-items: center !important;
+    display: grid;
+    justify-content: flex-start;
+    padding-top: 0 !important;
+
+    > img {
```

Comment:
Ideally, this would target a class. Allowing someone to change the element without braking the styling. For example, in the future we might switch from `img` element to `picture`.

Reply:
Ahh ok will do, thank you!